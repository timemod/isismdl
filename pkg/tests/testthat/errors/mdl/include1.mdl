ident a = 1; 

#include "incl/incl_err.mdl"
