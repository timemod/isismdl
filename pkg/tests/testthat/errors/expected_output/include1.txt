Variable a used more than once in left hand side
Line: 1 of file mdl/incl/incl_err.mdl
a = 1;
  ^

Duplicate equation a
Line: 1 of file mdl/incl/incl_err.mdl
a = 1;
     ^

end; statement not allowed in included file
Line: 3 of file mdl/incl/incl_err.mdl
end;
   ^

